src/framework/audio/internal/iaudiodriver.h

```diff
@@ -55,6 +57,11 @@ class IAudioDriver : MODULE_EXPORT_INTERFACE
     virtual bool open(const Spec& spec, Spec* activeSpec) = 0;
     virtual void close() = 0;
     virtual bool isOpened() const = 0;
+
+    virtual std::string device() const = 0;
+    virtual bool selectDevice(std::string name) = 0;
+    virtual std::vector<std::string> availableDevices() const = 0;
+    virtual async::Notification deviceListChanged() const = 0;
```

Comment:
Slightly inconsistent names. It is better that the names are corresponding -  `someThing()` and `someThingChanged()`
i.e. `availableDevices()` and `availableDevicesChanged()` or `deviceList()` and `deviceListChanged()`

Reply:
fixed